MatrixSDK/Data/MXRoomLastMessage.m

```diff
@@ -58,6 +59,19 @@ - (instancetype)initWithEvent:(MXEvent *)event
     return self;
 }
 
+- (nullable NSData*)encryptedAttributedString
+{
+    if (_attributedText)
+    {
+        NSData* archieved = [NSKeyedArchiver archivedDataWithRootObject:_attributedText];
```

Comment:
Typo `archieved` => `archived`.

Also `archivedDataWithRootObject` is now deprecated, so best to use the recommended variant with explicit error handling.

Reply:
Applied -> https://github.com/matrix-org/matrix-ios-sdk/pull/1718/commits/1e1c24323531ced30bc9d7bfab5e3b1b67abb822